deploy-grafana.sh

```diff
@@ -15,11 +15,12 @@ docker run --detach \
     --restart=always \
     --cpus=1 \
     --memory=1g \
-    -p 0.0.0.0:3000:3000 \
+    -p 0.0.0.0:3370:3370 \
+    -e GF_SERVER_SERVE_FROM_SUB_PATH=true \
```

Comment:
Is there a reason this can't / shouldn't be the default value in our image?

Reply:
My bad :-)